nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/validator/DocumentConstraintValidator.java

```diff
@@ -130,7 +132,11 @@ private boolean isResolvable(ValueReference ref, ValueExpression ve) {
     protected List<ConstraintViolation> doValidate(FacesContext context, ValueReference vref, ValueExpression e,
             Object value) {
         DocumentValidationService s = Framework.getService(DocumentValidationService.class);
+        // document validation
         DocumentValidationReport report = null;
+        if (!s.isActivated(CTX_JSFVALIDATOR, null)) {
```

Comment:
What do you think about using a more explicit variable name for the DocumentValidationService `s`?